setup.py

```diff
@@ -16,7 +16,7 @@
     packages=find_packages(exclude=('tests.*', 'tests', 'example')),
     install_requires=[
         'Django>=1.8',
-        'sqlparse',
+        'sqlparse>=0.2.0',
```

Comment:
Since we don't set upper limits, how could our requirements get out of date?

(Also I'm not a fan of cargo-culting badges.)